ipfs.go

```diff
@@ -37,12 +36,6 @@ import (
 	"github.com/multiformats/go-multihash"
 )
 
-func init() {
-	ipld.Register(cid.DagProtobuf, merkledag.DecodeProtobufBlock)
-	ipld.Register(cid.Raw, merkledag.DecodeRawBlock)
-	ipld.Register(cid.DagCBOR, cbor.DecodeBlock) // need to decode CBOR
-}
```

Comment:
These appear unneeded, is there something I'm missing here?

Reply:
I'm relatively confident that without these you would do a ipfslite.DagService.Get() and the thing would error because it didn't know how to parse stuff (particularly with CBOR).

Perhaps this is no longer the case, perhaps these are preregistered somewhere else, I don't know. It was a way to not make the users go through the hassle of figuring out that they have to register the codecs they want to use.

Reply:
So the merkledag library will still have a global registry for now https://github.com/ipfs/boxo/pull/322 that will use the global go-ipld-prime registry + some overrides for dag-pb and raw which means the dagservice will be fine.

I don't think a breaking change to go-merkledag is in the cards for this round of PRs, but if you'd like the registry used by merkledag to be configurable I'm cool exposing it as a public global with sufficient warnings around it.